presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/RuleAssert.java

```diff
@@ -36,40 +36,37 @@
 
 public class RuleAssert
 {
-    private final Metadata metadata;
-    private Session session;
     private final Rule rule;
 
     private final PlanNodeIdAllocator idAllocator = new PlanNodeIdAllocator();
+    private final LocalQueryRunner queryRunner;
```

Comment:
I was trying to prevent having constantly increasing numbers of arguments to the RuleAssert all of which come from the LocalQueryRunner (e.g. eventually stats and cost calculation too).  But I see the value of keeping it independent. I'll change it.